dist/debian/control.template

```diff
@@ -12,13 +12,13 @@ Architecture: any
 Description: Scylla database main configuration file
  Scylla is a highly scalable, eventually consistent, distributed,
  partitioned row DB.
-Replaces: %{product}-server (<< 1.1)
+Replaces: %{product}-server (<< 1.1), %{product}-enterprise-server (<<2024.2)
 Conflicts: %{product}-server (<< 1.1)
 
 Package: %{product}-server
 Architecture: any
 Depends: ${misc:Depends}, %{product}-conf (= ${binary:Version}), %{product}-python3 (= ${binary:Version})
-Replaces: %{product}-tools (<<5.5)
+Replaces: %{product}-tools (<<5.5), %{product}-enterprise-tools (<<2024.2), %{product}-enterprise-python3 (<<2024.2), %{product}-enterprise-cqlsh (<<2024.2)
```

Comment:
why does the new scylla-server package "replaces"  scylla-enterprise-tool? BTW, i don't think it makes sense to use "${product}-enterprise" here, "scylla-enterprise" is a product name. and we are replacing the packages with these *fixed* names, not a set of packages whose name can be generated from this product -- "scylla".